examples/redshift/README.md

```diff
@@ -0,0 +1,32 @@
+# What is this?
+
+This example shows how one can use Opta combined with existing remote terraform modules to spin up
```

Comment:
suggestion: _This example shows how Opta can use a remote terraform module to spin up an Amazon Redshift_

in my first read, I found the _existing_ word confusing with a user having existing terraform stack.